spt.c

```diff
@@ -80,16 +80,28 @@ void
 remaining_time(int sigint)
 {
 	char buf[17];
-	if (signal(SIGUSR1, SIG_IGN) != SIG_IGN)
-		signal(SIGUSR1, remaining_time);
+	//if (signal(SIGUSR1, SIG_IGN) != SIG_IGN)
+	//	signal(SIGUSR1, remaining_time);
```

Comment:
If it's commented out, IIRC it only handle the signal `SIGUSR1` once which means it only display the remaining time once. Okay, I think the spaces below aren't necessary.